File: packages/internal/generators-files/files/boot/application/files/java/src/main/java/__packageDirectory__/__appClassName__.java
package <%= packageName %>;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication<% if(!minimal) { %>(scanBasePackages = "<%= groupId %>")<% } %>
public class <%= appClassName %> {

  public static void main(String[] args) {
    SpringApplication.run(<%= appClassName %>.class, args);
  }

}

File: packages/internal/generators-files/boot/application/files/kotlin/src/main/kotlin/__packageDirectory__/__appClassName__.kt
package <%= packageName %>

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication<% if(!minimal) { %>(scanBasePackages = arrayOf("<%= groupId %>"))<% } %>
class <%= appClassName %>

fun main(args: Array<String>) {
  runApplication<<%= appClassName %>>(*args)
}


